magpie-core/src/main/java/io/openraven/magpie/core/cspm/services/PolicyAnalyzerServiceImpl.java

```diff
@@ -53,52 +53,61 @@ public void init(MagpieConfig config) {
     List<Violation> violations = new ArrayList<>();
 
     policies.forEach(policy -> {
-      LOGGER.info("Analyzing policy - {}", policy.getPolicy().getName());
-      policy.getPolicy().getRules().forEach(rule -> {
-        if (rule.isManualControl()) {
-          LOGGER.info("Rule not analyzed (manually controlled) - {}", rule.getName());
-        } else {
-          LOGGER.info("Analyzing rule - {}", rule.getName());
-          LocalDateTime evaluatedAt = LocalDateTime.now();
+        if (policy.getPolicy().isEnabled()) {
```

Comment:
This would look cleaner if you did
```java
final var p = policy.getPolicy();
```
and then referenced that rather than calling `policy.getPolicy()` throughout the following lines.

Reply:
Done :)